src/platform/elements/value/Render.ts

```diff
@@ -217,9 +217,14 @@ export class RenderPipe implements PipeTransform {
                 break;
             case 'Opportunity':
             case 'JobOrder':
-            case 'Placement':
                 text = value.label || value.title || '';
                 break;
+            case 'Placement':
+                text = `${value.candidate.firstNames || ''} ${value.candidate.lastName || ''}`;
+                if (value.jobOrder) {
+                    text = `${text} - ${value.jobOrder.title}`;
+                }
```

Comment:
please add some defensive checks - `value.candidate` may be undefined. also it should be firstName not firstNames